Assets/_Project/Scripts/Singletons/AudioManager.cs

```diff
@@ -0,0 +1,90 @@
+using Editarrr.Misc;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Editarrr.Audio
+{
+
+
+    public class AudioManager : MonoBehaviour
+    {
+        public static AudioManager instance;
```

Comment:
This is not a way I've seen audio done before :) I can see that we've got a Pool of audio sources on the prefab and you are cycling through them to in theory be able to play several sounds at the same time or in close succession. Nice work!